src/api_manager/context/request_context.cc

```diff
@@ -73,6 +76,22 @@ std::string GenerateUUID() {
   return uuid_buf;
 }
 
+inline std::vector<std::string> &split(const std::string &s, char delim,
+                                       std::vector<std::string> &elems) {
```

Comment:
why are they two places to return output,  1) function return and 2) second argument

per style guide,   if output in the argument, it has to be the last,  and it has to use raw pointer

why return a vector<>&,   a reference

Reply:
Changed.